### DISCUSSION TITLE
问题

### DISCUSSION THREAD

**Discussion by ZaiZHaik** [Meloong-Git/PCL#3764] [提问] [answered]
如何手动将Minecraft缓存更改至其他盘

**Comment by MoYuan-CN** [comment, marked as answer]
是这个吗
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/35274f5e-d4b1-4cac-aadb-ed9aa84e7f54)

**Reply by ZaiZHaik** [reply]
Thank you
